go/vt/vtctl/vtctl.go

```diff
@@ -3110,7 +3112,7 @@ func commandApplySchema(ctx context.Context, wr *wrangler.Wrangler, subFlags *fl
 		AllowLongUnavailability: *allowLongUnavailability,
 		DdlStrategy:             *ddlStrategy,
 		Sql:                     parts,
-		SkipPreflight:           *skipPreflight,
+		SkipPreflight:           true,
```

Comment:
Perhaps we should be deprecating the protobuf field for this as well? I'm not certain about that though - whether we do this now and deprecate the protobuf field in the next release.
@ajm188 what do you suggest?

Reply:
i know i just said "yes" on the other PR, but i think we need to wait one more cycle, since old tablets will be consuming this field, which will take the zero value (`false`) instead of what we unconditionally pass here (`true`).

so deprecate flag + always pass `true` (v15) => remove flag + deprecate protobuf field + tablets ignore value and always skip-preflight (v16)